dsp-repository/v1/dsp-repository.shacl.ttl

```diff
@@ -26,32 +26,670 @@
+        sh:datatype xsd:string ;
+        sh:minCount 1 ;
+    ] ;
+    sh:closed true ;
+    sh:ignoredProperties (rdf:type) . # TODO: check, if there is a more appropriate usage for this
+
+### --- Shape of dsp-repo:Organization ---
+dsp-repo:OrganizationShape
+    a sh:NodeShape ;
+    sh:targetClass dsp-repo:Organization ;
+    sh:property [
+        sh:path dsp-repo:hasIdentifier ;
+        sh:name "Identifier" ;
+        sh:description "Identifier of the organization from https://www.grid.ac/"@en ;
+        sh:class schema:URL ;
+        sh:maxCount 1 ;
+    ] ;
+    sh:property [
+        sh:path dsp-repo:hasName ;
+        sh:name "Legal name" ;
+        sh:description "Legal name of the organization"@en ;
+        sh:datatype xsd:string ;
+        sh:minCount 1 ;
+    ] ;
+    sh:property [
+        sh:path dsp-repo:hasEmail ;
+        sh:name "Organization e-mail" ;
+        sh:description "E-mail address of the organization"@en ;
+        sh:datatype xsd:string ;
+        sh:maxCount 1 ;
+    ] ;
+    sh:property [
+        sh:path dsp-repo:hasAddress ;
+        sh:name "Address" ;
+        sh:description "Postal address of the organization"@en ;
+        sh:datatype xsd:string ;
+        sh:maxCount 1 ;
+    ] ;
+    sh:property [
+        sh:path dsp-repo:hasURL ;
+        sh:name "URL of the organization" ;
+        sh:description "URL"@en ;
+        sh:class schema:URL ;
+        sh:maxCount 1 ;
+    ] ;
+    sh:closed true ;
+    sh:ignoredProperties (rdf:type) .
+
+### --- Shape of dsp-repo:Grant ---
+dsp-repo:GrantShape
+    a sh:NodeShape ;
+    sh:targetClass dsp-repo:Grant ;
+    sh:property [
+        sh:path dsp-repo:hasName ;
+        sh:name "Name" ;
+        sh:description "Name of the grant [Ambizione, etc.]"@en ;
+        sh:datatype xsd:string ;
+        sh:maxCount 1 ;
+    ] ;
+    sh:property [
+        sh:path dsp-repo:hasURL ;
+        sh:name "URL" ;
+        sh:description "URL of the grant"@en ;
+        sh:class schema:URL ;
+        sh:maxCount 1 ;
+    ] ; 
+     sh:property [
+        sh:path dsp-repo:hasNumber ;
+        sh:name "Number" ;
+        sh:description "Number of the grant"@en ;
+        sh:datatype xsd:string ;
+        sh:maxCount 1 ;
+    ] ; 
+     sh:property [
+        sh:path dsp-repo:hasFunder ;
+        sh:name "Funder" ;
+        sh:description "Funder of the grant"@en ;
+        sh:or( [ sh:class dsp-repo:Person ] [ sh:class dsp-repo:Organization ] ) ;
+        sh:maxCount 1 ;
+    ] ; 
+    sh:closed true ;
+    sh:ignoredProperties (rdf:type) .
+
+### --- Shape of dsp-repo:DataManagementPlan ---
+dsp-repo:DataManagementPlanShape
+    a sh:NodeShape ;
+    sh:targetClass dsp-repo:DataManagementPlan ;
+    sh:property [
+        sh:path dsp-repo:hasURL ;
+        sh:name "URL" ;
+        sh:description "URL of the data management plan"@en ;
+        sh:class schema:URL ;
+        sh:maxCount 1 ;
+    ] ;
+    sh:property [
+        sh:path dsp-repo:hasDescription ;
```

Comment:
I feel it strange to use `dsp-repo:hasDescription ;` for what would be a boolean or a controlled vocabulary. I would create a new property `dsp-repo:hasAvailability ;` with  a `sh:datatype xsd:string ;` constraint and, like Balduin suggested, at some point, 

> we could use sh:in to provide a list of possible values.
(see https://www.w3.org/TR/shacl/#InConstraintComponent )

Reply:
We can change the property to `dsp-repo:hasAvailability`.

Reply:
maybe `isAvailable`?

Reply:
changed to `isAvailable`